cmd/keeper/keeper.go

```diff
@@ -89,6 +89,7 @@ type config struct {
 	pgSUPasswordFile        string
 	pgInitialSUUsername     string
 	pgInitialSUPasswordFile string
+	defaultReplAccess       string
```

Comment:
Sorry, I meant to set it as a cluster spec option, so it could be set globally: https://github.com/sorintlab/stolon/blob/master/pkg/cluster/cluster.go#L186

Reply:
Not a problem I'll change that

Reply:
Changes done